### PR TITLE
CC-17227 Restrict access to merchant portal for main merchant user

### PR DESCRIPTION
**Pull request by ievgen-malykhin**
## PR Description

Added documentation for [CC-17227](https://spryker.atlassian.net/browse/CC-17227)
Documentation task: https://spryker.atlassian.net/browse/CC-20160

## Checklist
- [X] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/user/features/202108.0/marketplace-merchant-feature-overview/main-merchant-concept.md [hunk 1]
```diff
@@ -33,3 +33,9 @@ The workflow of the main merchant state machine is schematically displayed in th
 If the [Marketplace Return Management](/docs/marketplace/user/features/{{page.version}}/marketplace-return-management-feature-overview.html) feature is integrated into the project, the main merchant state machine obtains an additional return subprocess, and the flow looks like this:
 
 ![img](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+Merchant/Main+merchant+concept/marketplace-main-merchant-return-process.png)
+
+## Main merchant user
+
+Main merchant user is a user which has assigned `root_group`.
```
[the remark this answers is not in the document]
Comment (ievgen-malykhin) [reply docs/marketplace/user/features/202108.0/marketplace-merchant-feature-overview/main-merchant-concept.md:39]: Yes, it definitely sounds better!

File: docs/marketplace/user/merchant-portal-user-guides/202108.0/logging-in-to-the-merchant-portal.md [hunk 2]
```diff
@@ -14,6 +14,12 @@ To log in to the Merchant Portal, a [merchant user](/docs/marketplace/user/featu
 
 Each section in this article contains reference information. Make sure to review it before you start, or look up the necessary information as you go through the process.
 
+{% info_block warningBox "Note" %}
+
+Merchant portal login is restricted for the [Main merchant user](/docs/marketplace/user/features/{{page.version}}/marketplace-merchant-feature-overview/main-merchant-concept.html#main-merchant-user).
```
[the remark this answers is not in the document]
Comment (ievgen-malykhin) [reply docs/marketplace/user/merchant-portal-user-guides/202108.0/logging-in-to-the-merchant-portal.md]: Fixed

**Comment by dmytro-sokolovskyi** [comment]
Please, fix validation errors.

**Comment by ievgen-malykhin** [comment]
> Please, fix validation errors.

Fixed

**Review by dmytro-sokolovskyi — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**